db/migrate/20170821170916_add_nvra_index_to_installed_packages.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToInstalledPackages < ActiveRecord::Migration
```

Comment:
I had to change this to `AddNvraIndexToInstalledPackages`, it was unhappy and got confused otherwise